conf/messages.cy

```diff
@@ -2,15 +2,39 @@ service.name = Welsh: External guidance
 hello.world = Helo o''r cais view-external-guide-frontend!
 hello.world.currentLanguage = Y gosodiad iaith cyfredol yw "{0}",
 
+service.phaseName = beta
```

Comment:
Where possible I'd use the translations from the coronavirus project.

Reply:
all in